addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -72,6 +64,27 @@ private _turretPaths = ((fullCrew [_vehicle, "gunner", true]) + (fullCrew [_vehi
         _action = [_name, _text, _icon, _statement, _condition, {}, [_hitpoint], _position, 2] call EFUNC(interact_menu,createAction);
         [_type, 0, [], _action] call EFUNC(interact_menu,addActionToClass);
 
+        // Create a wheel interaction
+        private _root = format ["Wheel_%1_%2", _forEachIndex, _hitpoint];
+        private _action = [_root, localize LSTRING(SpareWheel), ["","#FFFFFF"], {}, {true}, {}, [_hitpoint], _position, 2, nil, LINKFUNC(modifySelectionInteraction)] call EFUNC(interact_menu,createAction);
```

Comment:
```suggestion
        private _action = [_root, localize "str_xbox_wheel", ["","#FFFFFF"], {}, {true}, {}, [_hitpoint], _position, 2, nil, LINKFUNC(modifySelectionInteraction)] call EFUNC(interact_menu,createAction);
```
"Spare Wheel" is a little funny for a wheel on a vic, I think just "wheel" would be better